### ISSUE TITLE
contractions

### ISSUE THREAD

**Issue by RobinHankin** [RobinHankin/stokes#23]
Weintraub defines a _contraction_ as follows.  Given a k-form ```\phi```, with k>1, then ```\phi_v```,  the contraction of phi with v is given by

```
\phi_v\left(v^1,\ldots,v^{k-1}\right)= \phi\left(v,v^1,\ldots,v^{k-1}\right)
```

**Comment by RobinHankin** [comment]
[f.pdf](https://github.com/RobinHankin/wedge/files/3440924/f.pdf)

**Comment by RobinHankin** [comment]
Reference is Steven H. Weintraub 2014.  "Differential forms: theory and practice", Elsevier.  Contractions defined in  Definition 2.2.23 in chapter 2, page 77.

**Comment by RobinHankin** [comment]
fixed in aa68360
